### PR TITLE
Updated delete_event method in data.py

### PR DESCRIPTION
**Pull request by aditya1702**

### CODE REVIEW

**Comment by aditya1702** [comment]
@SaptakS  I know you have already made this commit but you have done it along with the review commits. I think we can merge this event deletion commit first so that the checks are passed. Are you ok with it?

File: open_event/helpers/data.py [hunk 1]
```diff
@@ -461,6 +461,11 @@ def update_event(form, event):
     @staticmethod
     def delete_event(e_id):
         EventsUsers.query.filter_by(event_id=e_id).delete()
+        Sponsor.query.filter_by(event_id=e_id).delete();
+        Speaker.query.filter_by(event_id=e_id).delete();
```
Comment (juslee) [inline open_event/helpers/data.py]: Codacy Issue found: [Unnecessary semicolon](https://www.codacy.com/app/triplez/open-event-orga-server/file/2411608151/issues/source?bid=3233864&fileBranchId=3235400#l465)

File: open_event/helpers/data.py [hunk 2]
```diff
@@ -461,6 +461,11 @@ def update_event(form, event):
...
+        Track.query.filter_by(event_id=e_id).delete();
+        Microlocation.query.filter_by(event_id=e_id).delete();
```
Comment (juslee) [inline open_event/helpers/data.py]: Codacy Issue found: [Unnecessary semicolon](https://www.codacy.com/app/triplez/open-event-orga-server/file/2411608151/issues/source?bid=3233864&fileBranchId=3235400#l467)

File: open_event/helpers/data.py [hunk 3]
```diff
@@ -461,6 +461,11 @@ def update_event(form, event):
...
+        Session.query.filter_by(event_id=e_id).delete();
```
Comment (juslee) [inline open_event/helpers/data.py]: Codacy Issue found: [Unnecessary semicolon](https://www.codacy.com/app/triplez/open-event-orga-server/file/2411608151/issues/source?bid=3233864&fileBranchId=3235400#l468)

File: open_event/helpers/data.py [hunk 4]
```diff
@@ -461,6 +461,11 @@ def update_event(form, event):
     @staticmethod
     def delete_event(e_id):
         EventsUsers.query.filter_by(event_id=e_id).delete()
+        Sponsor.query.filter_by(event_id=e_id).delete();
```
Comment (juslee) [inline open_event/helpers/data.py]: Codacy Issue found: [Unnecessary semicolon](https://www.codacy.com/app/triplez/open-event-orga-server/file/2411608151/issues/source?bid=3233864&fileBranchId=3235400#l464)

File: open_event/helpers/data.py [hunk 5]
```diff
@@ -461,6 +461,11 @@ def update_event(form, event):
...
+        Speaker.query.filter_by(event_id=e_id).delete();
+        Track.query.filter_by(event_id=e_id).delete();
```
Comment (juslee) [inline open_event/helpers/data.py]: Codacy Issue found: [Unnecessary semicolon](https://www.codacy.com/app/triplez/open-event-orga-server/file/2411608151/issues/source?bid=3233864&fileBranchId=3235400#l466)

File: open_event/helpers/data.py [hunk 6]
```diff
@@ -461,6 +461,11 @@ def update_event(form, event):
     @staticmethod
     def delete_event(e_id):
         EventsUsers.query.filter_by(event_id=e_id).delete()
+        Sponsor.query.filter_by(event_id=e_id).delete()
+        Speaker.query.filter_by(event_id=e_id).delete()
+        Track.query.filter_by(event_id=e_id).delete()
+        Microlocation.query.filter_by(event_id=e_id).delete()
```
Comment (juslee) [inline open_event/helpers/data.py]: Codacy Issue found: [Unnecessary semicolon](https://www.codacy.com/app/triplez/open-event-orga-server/file/2411617540/issues/source?bid=3233864&fileBranchId=3235400#l467)

File: open_event/helpers/data.py [hunk 7]
```diff
@@ -461,6 +461,11 @@ def update_event(form, event):
     @staticmethod
     def delete_event(e_id):
         EventsUsers.query.filter_by(event_id=e_id).delete()
+        Sponsor.query.filter_by(event_id=e_id).delete()
+        Speaker.query.filter_by(event_id=e_id).delete()
```
Comment (juslee) [inline open_event/helpers/data.py]: Codacy Issue found: [Unnecessary semicolon](https://www.codacy.com/app/triplez/open-event-orga-server/file/2411617540/issues/source?bid=3233864&fileBranchId=3235400#l465)

File: open_event/helpers/data.py [hunk 8]
```diff
@@ -461,6 +461,11 @@ def update_event(form, event):
...
+        Track.query.filter_by(event_id=e_id).delete()
+        Microlocation.query.filter_by(event_id=e_id).delete()
+        Session.query.filter_by(event_id=e_id).delete()
```
Comment (juslee) [inline open_event/helpers/data.py]: Codacy Issue found: [Unnecessary semicolon](https://www.codacy.com/app/triplez/open-event-orga-server/file/2411617540/issues/source?bid=3233864&fileBranchId=3235400#l468)

File: open_event/helpers/data.py [hunk 9]
```diff
@@ -461,6 +461,11 @@ def update_event(form, event):
     @staticmethod
     def delete_event(e_id):
         EventsUsers.query.filter_by(event_id=e_id).delete()
+        Sponsor.query.filter_by(event_id=e_id).delete()
```
Comment (juslee) [inline open_event/helpers/data.py]: Codacy Issue found: [Unnecessary semicolon](https://www.codacy.com/app/triplez/open-event-orga-server/file/2411617540/issues/source?bid=3233864&fileBranchId=3235400#l464)

File: open_event/helpers/data.py [hunk 10]
```diff
@@ -461,6 +461,11 @@ def update_event(form, event):
...
+        Speaker.query.filter_by(event_id=e_id).delete()
+        Track.query.filter_by(event_id=e_id).delete()
```
Comment (juslee) [inline open_event/helpers/data.py]: Codacy Issue found: [Unnecessary semicolon](https://www.codacy.com/app/triplez/open-event-orga-server/file/2411617540/issues/source?bid=3233864&fileBranchId=3235400#l466)

**Comment by juslee** [comment]
Update this PR with the latest master.

**Pull request closed without merging**